frontend/src/screens/dev/index.tsx

```diff
@@ -46,23 +49,16 @@ export default function Dev() {
         const userId = ev.currentTarget.dataset.userId
         ev.preventDefault()
         if (userId) {
-            await currentUser.become(userId)
+            await loginAsUser(userId)
+            await refetchEnvironment()
             nav('/')
```

Comment:
`refetchEnvironment()` doesn't actually do anything here, right?  they browser is navigating away from the page so the request will probably be canceled as it does, and the result will never be utilized.

Reply:
interestingly enough without refetchEnvironment we still have that issue where you can
- login as researcher
- logout
- login as user
- redirect to '/'
- you still see the researcher view

Reply:
i think refetch would be forcing the environment provider to update the state and propagate that down through the context